src/base.js

```diff
@@ -33,7 +33,7 @@ let config = new Config().merge({
   },
   output: {
     pathinfo: !inProduction,
-    path: 'build',
+    path: '/build',
```

Comment:
Absolute path breaks webpack production build, but `webpack-dev-server` works fine.
Please use the following to support both:
```
{
  path: process.cwd() + '/build',
}
```